#! /bin/sh -e

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -718,6 +718,8 @@
 if test "$Py_DEBUG" = 'true'
 then
 	DEBUG_EXT=_d
+else
+	DEBUG_EXT=
 fi
 AC_SUBST(DEBUG_EXT)
